app/code/VConnect/Erp/Api/Data/ExternalIdInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace VConnect\Erp\Api\Data;
+
+interface ExternalIdInterface
```

Comment:
is the purpose of this interface just provide a constants you need? an interface without any realisation is a bad idea. Especially when you put it in API/Data namespace. Does it anything in api?
 If you want to keep a class just for constant usage better to create a 'Config' class and specify constants there. For example: Vendor/ModuleName/Model/Config.php